library/src/main/java/com/datatorrent/lib/io/jms/JMSBase.java

```diff
@@ -143,7 +172,7 @@ public void setConnectionFactoryProperties(Map<String, String> connectionFactory
   @Deprecated
   public void setUser(String user)
```

Comment:
Good point. I will go ahead and ask dev & users about connectionFactoryClass. In the mean time will think and discuss about which option to go with (remove the convenience functions or mark them deprecated).